docs/reST/ref/image.rst

```diff
@@ -15,10 +15,11 @@ Note that there is no Image class; an image is loaded as a Surface object. The
 Surface class allows manipulation (drawing lines, setting pixels, capturing
 regions, etc.).
 
-The image module is a required dependency of pygame, but it only optionally
-supports any extended file formats. By default it can only load uncompressed
-``BMP`` images. When built with full image support, the ``pygame.image.load()``
-function can support the following formats.
+In the vast majority of installations, including those from pip, pygame is
+built to support extended formats with SDL2_Image. However, installations are
```

Comment:
Nice improvement mentioning that most installations will have extended formats supported.

IMHO "SDL2_Image" is jargon for most readers, and "pip" is noise here.

Reply:
I reworded it a bit.